jupyterlab_latex/build.py

```diff
@@ -178,6 +177,7 @@ def get(self, path = ''):
         # Parse the path into the base name and extension of the file
         tex_file_path = os.path.join(self.notebook_dir, path.strip('/'))
         tex_base_name, ext = os.path.splitext(os.path.basename(tex_file_path))
+		c = LatexConfig(config=self.config)
```

Comment:
Looks like an indentation error.

Reply:
Damn. Corrected it. I'm currently using Notepad++, which automatically uses tabs as indentation. Have to change that.